media/stylus/components/wiki/customcss.styl

```diff
@@ -909,23 +909,12 @@ dl {
 .indicatorInHeadline,
 .blockIndicator,
 .overheadIndicator {
-/*box-shadow: 1px 1px 1px #CCC*/
     margin: 0px 0px 1.5em;
     border: 5px solid;
-    padding: 10px 10px 0px 10px;
+    padding: 10px;
```

Comment:
These styles are being over-ridden by a declaration on line 948 and the inlineIndicators don't look right anymore. Before the added specificity of the redesign class was preventing this conflict.

I suspect but am not certain that we can just remove inlineIndicator from the other declaration.

Good testing page for this: https://developer.mozilla.org/en-US/docs/User:teoli/In_content